packages/block-library/src/terms-query/terms-query-content.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch } from '@wordpress/data';
+import { useInstanceId } from '@wordpress/compose';
+import { useEffect } from '@wordpress/element';
+import {
+	InspectorControls,
+	useBlockProps,
+	store as blockEditorStore,
+	useInnerBlocksProps,
+	privateApis as blockEditorPrivateApis,
+} from '@wordpress/block-editor';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../lock-unlock';
+import TermsQueryInspectorControls from './inspector-controls';
+
+const { HTMLElementControl } = unlock( blockEditorPrivateApis );
+
+export default function TermsQueryContent( {
+	attributes,
+	setAttributes,
+	clientId,
+	name,
+} ) {
+	const {
+		termQueryId,
+		termQuery = {},
+		tagName: TagName = 'div',
+	} = attributes;
+
+	const { __unstableMarkNextChangeAsNotPersistent } =
+		useDispatch( blockEditorStore );
+	const instanceId = useInstanceId( TermsQueryContent );
+	const blockProps = useBlockProps();
+	const innerBlocksProps = useInnerBlocksProps( blockProps );
+
+	const setQuery = ( newQuery ) => {
+		setAttributes( {
+			termQuery: {
+				...termQuery,
+				...newQuery,
+			},
+		} );
+	};
+
+	useEffect( () => {
+		if ( ! termQueryId ) {
+			__unstableMarkNextChangeAsNotPersistent();
+			setAttributes( { termQueryId: instanceId } );
+		}
+	}, [
+		termQueryId,
+		instanceId,
+		setAttributes,
+		__unstableMarkNextChangeAsNotPersistent,
+	] );
+
+	return (
+		<>
+			<InspectorControls>
```

Comment:
Why not put `InspectorControls` inside the `TermsQueryInspectorControls` component?